### ISSUE TITLE
Spirits of the Mothbinder (DP Cosmetic) is the wrong color.

### ISSUE THREAD

**Issue by Rhaeth** [ValveSoftware/Dota2-Gameplay#155]
### Cosmetic Item Name

Spirits of the Mothbinder

### Description

The 'Spirits of the Mothbinder' item for the 'Spirits' slot for Death Prophet from the Winter 2017 Treasure II has been the wrong color ever since the red version was added. The original version should be green, but right now both of them are red.

### Example Match ID

0

### Screenshots
Here is 'Spirits of the Mothbinder' in the loadout screen.
![image](https://user-images.githubusercontent.com/106772625/171779347-881f41cd-0e85-4037-ad4d-1ae13acbb606.png)
Here is 'Winter Lineage Spirits of the Mothbinder' in the loadout screen.
![image](https://user-images.githubusercontent.com/106772625/171779300-aebef5e8-ec83-4fa1-8d1a-069e4b3273df.png)
Here is how 'Spirits of the Mothbinder' currently looks ingame.
![image](https://user-images.githubusercontent.com/106772625/171779530-d7f64956-b408-4227-9619-881edb304de0.png)
Here is how it should look ingame. (Image taken from the wiki)
![image](https://user-images.githubusercontent.com/106772625/171779644-63bfa44d-75f0-48b1-90a2-b7215e775430.png)

**Comment by NMOTH** [comment]
Can you post screenshot?

**Comment by Rhaeth** [comment]
Yes, I can add screenshots of the current versions when I get home. The original version and the Winter Lineage (red) version can easily be viewed in game. They both appear to be using the same red version.
I don't think I have a screenshot of the original version saved but here is how the spirits should look, taken from the wiki.
![image](https://user-images.githubusercontent.com/106772625/171720794-5cbb72dd-ffd9-46cf-b0c1-ceb1fe58a40f.png)

**Comment by songib** [comment]
Yup, I post this issue today on [Reddit](https://www.reddit.com/r/DotA2/comments/wl99z5/spirits_of_the_mothbinder_bugged_spirit_model/), so I came here. Seems like not just me.

Alo Mr. Janitor at Valve, pls fix this. :(

**Comment by vydramain** [comment]
Have same issue but I have a correct Icon but incorrect ingame model:

file:///home/vydra/Pictures/Screenshots/Screenshot%20from%202023-01-15%2022-39-57.png

**Comment by bizzozeron** [comment]
Fixed in future update.
